ce/api/geo.py

```diff
@@ -67,14 +68,9 @@ def __init__(self, func, maxsize=50):
 
     def __call__(self, *args):
 
-        nc, wkt = args
-        # If we have no key, automatic cache miss
-        if (not hasattr(nc, 'model_id')):
-            log.debug('Cache MISS (attribute \'model_id\' not found)')
-            return self.func(*args)
+        fname, wkt, varname = args
 
-        # Set key to model_id and wkt polygon
-        key = (nc.model_id, wkt)
+        key = (fname, wkt)
```

Comment:
The netCDF4.Dataset object which *use to be* the parameter to `polygonToMask()` is no longer sufficient. `polygonToMask()` now has to open the file with both netCDF4 (to test for 0-360 grids) and `rasterio` to do the actual work.